### ISSUE TITLE
No endpoint for corporation fittings

### ISSUE THREAD

**Issue by Yantrio** [ccpgames/esi-issues#234]
I'd love to be able to get/post corp fittings, but there is currently no endpoint.

**Comment by markus-lassfolk** [comment]
Same. Being able to keep the Corp Fits up to date in an easy way, would be "priceless".

**Comment by testfax** [comment]
+1

**Comment by crossedxd** [comment]
Bit of a bump.  I'm working on a tool that compares fitted ships in corp assets against fittings to help with ship maintenance/upkeep, and being able to leverage the in-game corp fittings versus having to manually load each fitting into the tool would be a great help.

**Comment by sander2828** [comment]
agreed this endpoint would be real helpful and save tons of manual work.

**Comment by lunedis** [comment]
doesn't even need write access for a starter, just read access would already be useful.

**Comment by lizthegrey** [comment]
The "workaround" of importing to individual chars rather than to corp does exist, but has some risks. Someone may have a stale set of personal fittings imported from a shared fittings resource available to corp, and then accidentally buy a stale/outdated fit because they haven't synced their fittings recently. Or corps could regularly push updated fits to all of their members with API keys on file to avoid the out of date problem, but that's going to generate N times the traffic, where N is number of chars in corp, and I'm sure @HakShak does not want us making ESI fall over.

**Comment by chrfoyer** [comment]
This would be extremely useful in getting an overview of corp assets and current contracts. So because the in-backlog label was removed and not replaced with something else, does that mean they are no longer going to do it?
